### PR TITLE
Add gallery component doc

### PR DESCRIPTION
**Pull request by krampstudio**

### CODE REVIEW

File: docs/hub/_toctree.yml [hunk 1]
```diff
@@ -46,6 +46,8 @@
       title: Model Card Guidebook
     - local: model-card-landscape-analysis
       title: Landscape Analysis
+    - local: model-cards-components
+      title: Card Components
```
Comment (krampstudio) [inline docs/hub/_toctree.yml:50]: is there anything else needed to add a section to the ToC? It doesn't show on https://moon-ci-docs.huggingface.co/docs/hub/pr_1003/en/model-cards (cc @mishig25 )

File: docs/hub/model-cards-components.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Model Card components
+
+Model cards supports additionnal components you can place in your markdown.
```
Comment (gary149) [inline docs/hub/model-cards-components.md:3]:
```suggestion
**Model Card Components** are special elements that you can inject directly into your Model Card markdown to display powerful interface widgets directly in your model page. These components are authored by us, feel free to share ideas about new Model Card component in [this discussion](https://huggingface.co/spaces/huggingchat/chat-ui/discussions/312).
```

File: docs/hub/model-cards-components.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+TintinIA is fine-tuned version of Stable-Diffusion-xl trained on 125 comics panels from Tintin album. 
+
+```
+
+<div class="flex justify-center">
+<img width="650" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gallery.png"/>
+</div>
+
+The gallery uses the card's [widget metadata](/docs/hub/models-widgets-examples#text-to-image) to display the images and prompt.
```
Comment (gary149) [inline docs/hub/model-cards-components.md:24]:
```suggestion
The `<Gallery/>` component will use your Model Card [widget metadata](/docs/hub/models-widgets-examples#text-to-image) to display the images with each associated prompt.
```

File: docs/hub/model-cards-components.md (hunk 2 above)
Comment (julien-c) [reply docs/hub/model-cards-components.md:3]: not sure i'd say "widgets", might be confusing with the main widgets

"components" is ok no?

File: docs/hub/model-cards-components.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+<Gallery />
+
+## Model description
+
+TintinIA is fine-tuned version of Stable-Diffusion-xl trained on 125 comics panels from Tintin album. 
+
+```
+
+<div class="flex justify-center">
+<img width="650" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gallery.png"/>
```
Comment (osanseviero) [inline docs/hub/model-cards-components.md:21]: If possible also have darkmode

**Review by julien-c — COMMENTED** [review]
maybe hint that we would like to add this to a GUI editor at some point

File: docs/hub/_toctree.yml (hunk 1 above)
Comment (stefan-it) [reply docs/hub/_toctree.yml:50]: Hey @krampstudio ,

right link would be: https://moon-ci-docs.huggingface.co/docs/hub/pr_1093/en/model-cards

and there the Card Components section is shown 🎉

image

File: docs/hub/model-cards-components.md (hunk 4 above)
Comment (krampstudio) [reply docs/hub/model-cards-components.md]: see https://github.com/huggingface/hub-docs/pull/1093/commits/a26deb2f2df42fc393201128d2182c2ccdd71d0a

**Comment by krampstudio** [comment]
> maybe hint that we would like to add this to a GUI editor at some point

Something like https://github.com/huggingface/hub-docs/pull/1093/commits/e2ea2347c52e39b8b9bf44808889d4022b38d867 ?

**Comment by julien-c** [comment]
> Something like [e2ea234](https://github.com/huggingface/hub-docs/commit/e2ea2347c52e39b8b9bf44808889d4022b38d867) ?


Yes, i like it

**Review by gary149 — APPROVED** [review]

**Pull request merged**